dts/bindings/timer/st,stm32wb0-radio-timer.yaml

```
@@ -0,0 +1,28 @@
+# Copyright (c) 2025, STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: Radio timer properties for STM32WB0 family
+
+compatible: "st,stm32wb0-radio-timer"
+
+properties:
+  compensated-timer-freq:
+    type: int
+    required: true
+    enum:
+      - 409600
```

If there is only one value, it should be:
```suggestion
    const: 409600
```

Unless you plan to add more values when support for other clock sources are added? (but I would still go for `const:` now, and update to `enum:` later)